### PR TITLE
added error message to trigger when boolean flag is setoff (aka funct…

### PR DESCRIPTION
**Pull request by mtanca**
…ion cant computate)

@Bontgoy

### CODE REVIEW

File: calculator.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
-
 def calculator(first_number, second_number, operator)
-
+boolean_flag = true
```
Comment (Bontgoy) [inline calculator.rb]: This may not be needed. Consider just returning invalid input in the else statement below

**Comment by Bontgoy** [comment]
@mtanca these practice assessment and the assessment should not be on github.  They should be local repos only.  We keep them off github in order to keep the curriculum secure.  Please take this down after reviewing my comment.  Feel free to reach out in Slack if you have any questions about the material.

**Comment by mtanca** [comment]
Crap! Sorry Tyler. My first instinct is to always backup my work on GitHub and I wasn't aware during the challenge. I will delete it shortly.

Thanks,
Mark

**Comment by Bontgoy** [comment]
@mtanca no worries.  

Thank you!

**Pull request merged**